src/request.rs

```diff
@@ -277,6 +277,47 @@ impl RequestBuilder {
         Ok(self)
     }
 
+    /// Sends a multipart/formdata body.
+    ///
+    /// ```no_run
```

Comment:
By wrapping the example in similar stuff as the rest of the examples in the crate, the `no_run` can be removed, and all the `unwrap`s can be replaced with `?`s.

```
# fn run() -> Result<(), Box<::std::error::Error> {
let response = client.post("https://httpbin.org/post")?
    .multipart(...)
    .send()?;
# }
# fn main() {}
```